src/main/xcomponents/styledComponents2/Button.jsx

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
```

Comment:
Why make a new folder (`styledComponents2`)?

Reply:
so as to not modify previous stylesheet